frontend/admin/src/js/components/user/GeneralInformationTab/GeneralInfoTabDialogs.tsx

```diff
@@ -715,6 +716,11 @@ export const AddToPoolDialog: React.FC<{
                   if (currentPools.includes(pool.id)) {
                     return null;
                   }
+                  if (
+                    pool.advertisementStatus !== AdvertisementStatus.Published
+                  ) {
+                    return null;
+                  }
```

Comment:
We could likely do this with a single check:

```tsx
if (
  currentPools.includes(pool.id) ||
  pool.advertisementStatus !== AdvertisementStatus.Published
) {
  return null;
}
```

Reply:
oh that doesn't look messy at all, I thought it might have been messy to put into one comparison 
thanks
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4890/commits/4ef02138664f58b804abfb7333124991d4052977